lib/args.m4

```diff
@@ -22,11 +22,14 @@ Second-level command.
             - init
             - workspace
             - validate
+            - output
+            - output-json
             - plan
             - plan-local
             - apply
             - refresh
             - destroy
+            - destroy-local
```

Comment:
Shouldn't there really be two steps here: `plan-destroy` and `plan-destroy-local`? You're not actually destroying anything so it seems misleading to call it `destroy-*`. I'd kinda argue this isn't really necessary anyway since the `destroy` command is already interactive and will tell you what it's about to destroy. In the rare case that a destroy plan needs to be created the user can always run `make toolbox-bash` and drop into a bash shell.

Reply:
Renamed to `plan-destroy-local`